src/js/components/SelectMultiple/SelectMultiple.js

```diff
@@ -307,7 +307,8 @@ const SelectMultiple = forwardRef(
         if (optionIndexesInValue.length === 0) return '';
         if (optionIndexesInValue.length === 1)
           return applyKey(allOptions[optionIndexesInValue[0]], labelKey);
-        if (messages) return format({ id: 'select.multiple', messages });
+        if (messages)
+          return format({ id: 'selectMultiple.multiple', messages });
         return `${optionIndexesInValue.length} selected`;
```

Comment:
I think we should remove the 310 and 311.

Reply:
If we us `return format({
  id: 'selectMultipleNonTotal',
  values: {
    selectedCount: optionIndexesInValue.length,
    totalCount: allOptions.length,
  });`

instead of Line 311 ('selectMultiple.multiple'...) we cannot set our own text for the DefaultSelectTextInput. So it would be always the same text then the displayed text in the Dropdown.

Reply:
If the key "selectMultiple.multiple" and the "select.multiple" exist at the same time in the "default.json" it is still backward compatible? 
With two keys, however, two different messages can be displayed by default and since there are two different components, which is an advantage, isn't it?

Reply:
@MikeKingdom  
What do you mean by the aria text, is this the a11yTitle? 
If so, should it also be passed in the messageContext or should individual properties be created? 
If via messageContext then a separate key needs to be created for each title, right?

Reply:
In most cases it is a11yTitle. Sometimes there are ways to specify some aria related text like the aria-live content in SelectMultipleContainer.js.

No individual properties added. Separate message keys in default.json and any typescript/prop-type definitions for the `message` property for SelectMultiple. Ie. the typical way to localize an app is pass in a complete 'messages' property to grommet itself. The messages property for the SelectMultiple component itself is a way to override global messages for a specific instance of the component. 

And we should probably point out, after this PR is merged, the grommit site documentation for the component needs to be updated too for the `messages` property. The existing typescript/prop-types for SelectMultiple is picking up the definition of the `messages` prop from Select which may be a problem since they don't contain the same potential keys. This definition may need to be separate between Select and SelectMultiple to avoid annoying typescript and lint checkers

Reply:
I add the localization to all fields ad titles i have found in the select multiple files. 
I hope i found every thing in the files.